var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
No, my intention was to change which phases were run and hence it wouldn't make sense to directly reference 'install'. Instead I tried to set the phases run by trying to set phases directly. The first phase would then detect whether the proper variant was present and change the next step appropriately.